packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -988,6 +988,8 @@ const ruleset: any = {
       description: "This rule checks for references that aren't using latest version of common-types.",
       message: "{{error}}",
       severity: "warn",
+      disableForTypeSpec: true,
+      disableForTypeSpecReason: "TODO",
```

Comment:
@tjprescott @johanste @timotheeguerin if you folks could help me provide values for all `disableForTypeSpecReason: TODO`, that would be awesome :)

Reply:
I see there is a lot of duplicate rules in different ruleset, are they exactly the same?

Reply:
I can do that. Thank you @konrad-jamrozik!

Reply:
@timotheeguerin also - you folks were commenting on `.js` files instead of `.ts` files. `.js` files with rules are generated from `.ts` files by merging `common` ruleset with `arm` or `dataplan` ruleset. In the resulting `.js` files some rules are duplicated.